_posts/2017-08-02-July.md

```diff
@@ -0,0 +1,51 @@
+---
+layout: post
+title: "Dev Diary (July 2017)"
+author: "Corey Montella"
+tags: []
+---
+
+We've had quite a busy July! Let's get to it:
+
+## v0.4 milestones
+
+In July we started using v0.4 to build internal projects, in order to find bugs in the runtime and test the limits of its performance. If you've hit performance walls with the v0.3 runtime, you'll be happy to know the Rust runtime is several orders of magnitude faster. Beyond the runtime itself, we've started fleshing out the tooling:
+
+- Server - The sever works like it did in v0.2, hosting an Eve instance listening on a websocket. We also include a static file server that hosts the output of your program.
+
+- Browser Client - The browser client connects to a remote Eve server to which it streams records like key presses; and receives records like ticks of a timer. All the functionality from previous versions is still present -- no matter where Eve is running, the watcher system let's it interact with your browser (or anything else running JS or Rust) as if it were local.
```

Comment:
The example of what's exported is a bit confusing here. I'd say something like "HTML element records" or something.